packages/deposit-address/src/get-deposit-address/types.ts

```diff
@@ -14,9 +15,44 @@ export type SendOptions = {
   environment: Environment;
 };
 
-export type GetDepositAddressDependencies = {
+export type DepositAddressOptions = {
+  sourceChain: string;
+  destinationChain: string;
+  destinationAddress: string;
+  environment: Environment;
+  asset?: string; // if not specified, it is the native asset
```

Comment:
how about `string | 'native';`?

Reply:
this doesn't work since "native" is overridden by string

Reply:
how about we make `asset` to be required, not optional, but remove `getNativeWrapDepositAddress` from the `getDepositAddress` function?

So if people want to send native token, they have to use `getNativeWrapDepositAddress` instead.